src/risclog/sqlalchemy/db.py

```diff
@@ -77,12 +77,16 @@ def using_bind(self, name):
         return session
 
 
-def get_database(testing=False):
+def get_database(testing=False, keep_session=False, expire_on_commit=False):
```

Comment:
These values only seem to be implicitly tested. Is there a observable behaviour change, that can be converted to a testcase?